.github/scripts/trymerge.py

```diff
@@ -1020,7 +1035,8 @@ def merge(pr_num: int, repo: GitRepo,
           mandatory_only: bool = False,
           on_green: bool = False,
           land_checks: bool = False,
-          timeout_minutes: int = 400,
+          timeout_minutes: int = 350,
+          queue_timeout_minutes: int = 20,
```

Comment:
Do we anticipate that some jobs take some time to get triggered? If not, we could probably even have this be 10 or 5, no?

On the other end of the spectrum, for the XLA test that's barred on the build, would trying to merge the xla hash update fail sometimes since the test job wouldn't be triggered until the build is done?

Reply:
Hmm I'll try this out in canary I guess. I feel like this should work since we're only doing workflows now like pull so conditional jobs like that should still work.

Reply:
Not 100%, so the xla hash update specifies that the xla test is needed.